gui/src/app/StanSampler/StanSampler.ts

```diff
@@ -31,6 +31,7 @@ class StanSampler {
     #paramNames: string[] = [];
     #numChains: number = 0;
     #samplingStartTimeSec: number = 0;
+    #samplingOpts: SamplingOpts | undefined = undefined; // the sampling options used in the last sample call
```

Comment:
Now that we're storing numChains is redundant, correct?

I think we should set this to defaultSamplingOpts rather than undefined, and remove numChains

Reply:
Done.